Seneca-CDOT/telescope
# 3714: Upgraded post service node version from 14 to 16

## Issue This PR Addresses
Fixes #3714 


## Type of Change



## Description
- Upgrade node from 14 to 16 in post api service.


## Steps to test the PR
- Using steps in [setup telescope](https://telescope.cdot.systems/docs/getting-started/environment-setup):

1. Run `pnpm i` 
2. Run the services using `pnpm services:start`. Ensure all the containers are running
3. Ensure all posts are working correctly for the post service. 



## Checklist
